packages/frontend/core/src/components/page-list/docs/page-list-header.tsx

```diff
@@ -80,15 +87,41 @@ export const CollectionPageListHeader = ({
 
   const workspace = useService(Workspace);
   const { createEdgeless, createPage } = usePageHelper(workspace.docCollection);
+  const { openConfirmModal, closeConfirmModal } = useConfirmModal();
+
+  const createAndAddDocument = useCallback(
+    (createDocumentFn: () => Doc) => {
+      const newDoc = createDocumentFn();
+      collectionService.addPageToCollection(collection.id, newDoc.id);
+      closeConfirmModal();
```

Comment:
I think we don't need to close manually, it will close automatically by default.

Reply:
But in this case it doesn't work, so the close function needs to be added.

https://github.com/toeverything/AFFiNE/assets/102217452/4880f558-fc8b-4606-a040-713ba0d3bb42